schemas/2015-01-01/dtlArtifacts.json

```diff
@@ -14,14 +14,16 @@
     "properties": {
         "title": {
             "type": "string",
-            "description": "The title of your artifact."
+            "description": "The artifact's title."
         },
         "description": {
             "type": "string",
-            "description": "The description of your artifact."
+            "description": "The artifact's description."
         },
         "iconUri": {
-            "type": "string"
+            "type": "string",
+            "description": "An icon to identify and distinguish the artifact in a list. URL referencing the icon must use SSL.",
+            "pattern": "(^https:\/\/.*)|(^$)"
```

Comment:
You could try to simplify to `https:\/\/.*|^$` to avoid creating unnecessary groups.